onmt/utils/parse.py

```diff
@@ -320,4 +320,4 @@ def validate_train_opts(cls, opt):
 
     @classmethod
     def validate_translate_opts(cls, opt):
-        pass
+        opt.src_feats = eval(opt.src_feats) if opt.src_feats else {}
```

Comment:
Why do we need this line? What's the expected input for this argument?
I'm assuming it would be a list of file paths, then using `nargs` in the `group.add("-src_feats", ...)` of `onmt/opts.py` could do the trick.

Reply:
No, it's not a list of paths, it a dictionary mapping feature names with the corresponding file path. Like this:

`--src_feats "{'feat0': '../kk.txt.feats0', 'feat1': '../kk.txt.feats1'}"`

Reply:
Ok, that's reasonable then.